.gitignore

```diff
@@ -10,3 +10,4 @@ internal
 
 inst/doc
 
+.DS_Store
```

Comment:
Nit: I think it's better to normally exclude these files from a 'global' .gitignore, e.g. with

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c